pkg/s3wrapper/client.go

```diff
@@ -119,7 +119,7 @@ func (c *S3Client) IsAwsS3() bool {
 	if c.cfg.S3EndpointURL == "" || strings.HasSuffix(c.cfg.S3EndpointURL, awsEndpointSuffix) {
 		return true
 	}
-	return false
+	return true
```

Comment:
Was this change intended? This function always returns `true`.

Reply:
Wow good call